rules/windows/builtin/win_petitpotam_susp_tgt_request.yml

```diff
@@ -0,0 +1,34 @@
+title: PetitPotam Suspicious Kerberos TGT Request
+id: 6a53d871-682d-40b6-83e0-b7c1a6c4e3a5
+description: Detect suspicious Kerberos TGT requests. Once an attacer obtains a computer
+  certificate by abusing Active Directory Certificate Services in combination with
+  PetitPotam, the next step would be to leverage the certificate for malicious purposes.
+  One way of doing this is to request a Kerberos Ticket Granting Ticket using a tool
+  like Rubeus. This request will generate a 4768 event with some unusual fields depending
+  on the environment. This analytic will require tuning, we recommend filtering Account_Name
+  to the Domain Controller computer accounts.
+author: Mauricio Velazco, Michael Haag
+date: 2021/09/02
+references:
+    - https://github.com/topotam/PetitPotam
+    - https://isc.sans.edu/forums/diary/Active+Directory+Certificate+Services+ADCS+PKI+domain+admin+vulnerability/27668/
+    - https://github.com/splunk/security_content/blob/develop/detections/endpoint/petitpotam_suspicious_kerberos_tgt_request.yml
+tags:
+    - attack.credential_access
+    - attack.t1187
+logsource:
+    product: windows
+    service: security
+    definition: 'The advanced audit policy setting "Account Logon > Kerberos Authentication Service" must be configured for Success/Failure'
+detection:
+    selection:
+        EventID: 4768
+        Account_Name|endswith: '$'
+        Certificate_Thumbprint: '*'
+    filter_local:
+        Client_Address: '::1'
+        
+    condition: selection and not filter_local
```

Comment:
HI
use [original field name](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4768)
```yaml
        EventID: 4768
        TargetUserName|endswith: '$'
        CertThumbprint: '*'
    filter_local:
        IpAddress: '::1'
```
thanks

Reply:
thanks for reviewing it, I just pushed a new commit with the changes.